src/test/java/org/folio/spring/controller/TenantControllerIT.java

```diff
@@ -42,6 +45,21 @@ DataSource dataSource() {
     }
   }
 
+  @Test
+  void canCallManagementEndpointWithoutTenantHeader(
+      @Value("${management.endpoints.web.base-path:/admin}") String mgmtBasePath) throws Exception {
+    mockMvc.perform(get(mgmtBasePath + "/health"))
+      .andExpect(status().is(404)) // will be 200 if actuator is added
```

Comment:
Consider change `/health` to some url that is not provided by Spring Boot Actuator and is not likely to exist, and then remove `// will be 200 if actuator is added`

Reply:
You may have missed this - I asked in slack #folio-spring-base how people felt about incorporating actuator into folio-spring-base or some other implementation of GET /admin/health, a la RMB.  This comment was here as a placeholder pending the outcome of that discussion.

Regardless, I've cleaned this up.